src/style.css

```diff
@@ -1,3 +1,65 @@
+html {
+  font-family: Roboto, system-ui;
```

Comment:
Consider adding a fallback font-family to ensure text remains readable in case the primary font fails to load. For example, you could use a generic font family like 'sans-serif' as a fallback.